klippy/extras/display/display.cfg

```diff
@@ -13,7 +13,8 @@ text:
     # Show glyph
     {% if param_heater_name == "heater_bed" %}
       {% if heater.target %}
-        ~animated_bed~
+        {% set frame = (printer.toolhead.print_time|int % 2) + 1 %}
```

Comment:
This needs to be `printer.toolhead.estimated_print_time` - as print_time only updates during a print, while estimated_print_time is the host's estimate of the primary mcu's current clock in seconds.  Admittedly, estimated_print_time is not well named.